executive/Q5015503/current/popolo-m17n.json

```diff
@@ -1,22 +1,5 @@
 {
   "persons": [
-    {
-      "name": {
-        "lang:pt-br": "Floriano Peixoto",
-        "lang:pt": "Floriano Peixoto",
-        "lang:en": "Floriano Peixoto"
-      },
-      "id": "Q238688",
-      "identifiers": [
-        {
-          "scheme": "wikidata",
-          "identifier": "Q238688"
-        }
-      ],
-      "links": [
-
-      ]
-    },
     {
```

Comment:
Tancredo shouldn't be in here: https://github.com/everypolitician/commons-builder/issues/92